apps/passport-server/src/services/telegramService.ts

```diff
@@ -567,7 +666,52 @@ export class TelegramService {
     }
 
     this.authBot.command("help", helpResponse);
-    this.authBot.on("message", uwuResponse);
+    this.authBot.on("message", async (ctx) => {
+      logger(`[TELEGRAM] got message`, ctx.message);
+      const text = ctx.message.text;
+
+      if (isDirectMessage(ctx)) {
+        return ctx.reply(`TODO HELP MESSAGE`);
```

Comment:
?

Reply:
Need to put the help message when command not found here.

Reply:
Fixed!